### PR TITLE
Chore: Migrate AttachmentView to Typescript

### PR DESCRIPTION
**Pull request by reinaldonetof**
## How to test or reproduce

- You should be able to view files uploaded to your rooms
- Furthermore, check if the Attachment View

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/views/AttachmentView.tsx [hunk 1]
```diff
@@ -79,29 +98,17 @@ class AttachmentView extends React.Component {
...
 		const options = {
 			title,
-			headerLeft: () => (
-				<HeaderButton.CloseModal
-					testID='close-attachment-view'
-					navigation={navigation}
-					buttonStyle={{ color: themes[theme].previewTintColor }}
-				/>
-			),
+			headerLeft: () => <HeaderButton.CloseModal testID='close-attachment-view' navigation={navigation} />,
```
Comment (AlexAlexandre) [inline app/views/AttachmentView.tsx:101]: why remove the buttonStyle ?

File: app/views/AttachmentView.tsx [hunk 2]
```diff
@@ -79,29 +98,17 @@ class AttachmentView extends React.Component {
...
 			headerRight: () =>
-				Allow_Save_Media_to_Gallery ? (
-					<HeaderButton.Download
-						testID='save-image'
-						onPress={this.handleSave}
-						buttonStyle={{ color: themes[theme].previewTintColor }}
-					/>
-				) : null,
+				Allow_Save_Media_to_Gallery ? <HeaderButton.Download testID='save-image' onPress={this.handleSave} /> : null,
```
Comment (AlexAlexandre) [inline app/views/AttachmentView.tsx:103]: same question

File: app/views/AttachmentView.tsx [hunk 3]
```diff
@@ -79,29 +98,17 @@ class AttachmentView extends React.Component {
...
 			headerBackground: () => <View style={{ flex: 1, backgroundColor: themes[theme].previewBackground }} />,
 			headerTintColor: themes[theme].previewTintColor,
 			headerTitleStyle: { color: themes[theme].previewTintColor, marginHorizontal: 10 }
 		};
 		navigation.setOptions(options);
 	};
 
-	getVideoRef = ref => (this.videoRef = ref);
+	getVideoRef = (ref: Video | null) => (this.videoRef = ref);
```
Comment (AlexAlexandre) [inline app/views/AttachmentView.tsx:111]: why this `null` ?

File: app/views/AttachmentView.tsx (hunk 1 above)
Comment (reinaldonetof) [reply app/views/AttachmentView.tsx:101]: There isn't buttonStyle when looking into the component

**Review by AlexAlexandre — APPROVED** [review]
LGTM :)

**Comment by AlexAlexandre** [comment]
For everyone's knowledge, here is a link about changing the .ios.tsx and .android.tsx files

https://stackoverflow.com/a/59972215

#3459 (comment)

**Comment by AlexAlexandre** [comment]
@reinaldonetof  resolve the conflict, please. 
Thanks  :)

**Pull request merged**